.github/workflows/superset-python-integrationtest.yml

```diff
@@ -150,7 +150,9 @@ jobs:
       SUPERSET_CONFIG: tests.integration_tests.superset_test_config
       REDIS_PORT: 16379
       SUPERSET__SQLALCHEMY_DATABASE_URI: |
-        sqlite:///${{ github.workspace }}/.temp/unittest.db
+        sqlite:///${{ github.workspace }}/.temp/superset.db?check_same_thread=true
+      SUPERSET__SQLALCHEMY_EXAMPLES_URI: |
```

Comment:
Per [here](https://github.com/apache/superset/blob/9033e72679fad7efdabc0f5b28607d974a7b5e1f/tests/integration_tests/superset_test_config.py#L46-L48), if undefined the metadata and example databases are the same and given that SQLite doesn't support schemas. It seems prudent to split these into two after @dpgaspar's https://github.com/apache/superset/pull/25003 which isolated the examples database.